recipe/meta.yaml

```diff
@@ -35,7 +32,6 @@ requirements:
   run:
     - python
     - blosc
-    - numpy
```

Comment:
ref: https://github.com/Blosc/python-blosc/blob/v1.8.3/RELEASE_NOTES.rst#changes-from-161-to-162